platform/android/src/test/render_test_runner.cpp

```diff
@@ -41,17 +42,178 @@ void Log::platformRecord(EventSeverity severity, const std::string& msg) {
 
 } // namespace mbgl
 
+namespace {
+std::string jstring2string(JNIEnv* env, jstring jStr) {
+    if (!jStr) {
+        return "";
+    }
+    const jclass stringClass = env->GetObjectClass(jStr);
+    const jmethodID getBytes = env->GetMethodID(stringClass, "getBytes", "(Ljava/lang/String;)[B");
+    const jbyteArray stringJbytes = (jbyteArray)env->CallObjectMethod(jStr, getBytes, env->NewStringUTF("UTF-8"));
+
+    size_t length = (size_t)env->GetArrayLength(stringJbytes);
+    jbyte* pBytes = env->GetByteArrayElements(stringJbytes, NULL);
+
+    std::string ret = std::string((char*)pBytes, length);
+    env->ReleaseByteArrayElements(stringJbytes, pBytes, JNI_ABORT);
+
+    env->DeleteLocalRef(stringJbytes);
+    env->DeleteLocalRef(stringClass);
```

Comment:
Let's use smart pointers instead of manual freeing, afaik jni has `UniqueLocalRef` for this purpose

Reply:
@mapbox/core-sdk , can we encapsulate `jobject` within a unique_ptr that having `DeleteLocalRef` as a deleter?